docs/cross-chain-payments-and-execution.md

```diff
@@ -56,6 +56,53 @@ A UserOp can contain multiple `ExecuteChainInfo`s, meaning you can have "atomic"
 
 Since a UserOp can contain multiple `PaymentChainInfo`s and `ExecuteChainInfos`, this approach can be used for multi-hop execution or payments. We just require the `owner` and `intermediary` of every BridgeWallet involved to sign the UserOp.
 
+## Example with 2 hops
+Alice has funds on chainA (blue) with intermediary Irene. Irene has funds on chainB (green) with intermediary Isaac. Isaac has funds on chainC (red).
+
+Alice wants to execute a transaction on chainC. She crafts the transaction but does not yet sign it. Instead, she routes it to Irene, who appends a payment for herself in the Alice-Irene wallet on chainA. Irene forwards it to Isaac, who appends a payment for himself on the Irene-Isaac wallet on chainB. This completes the chain. Isaac combines the two payments and the transaction into a `UserOp`. This is then countersigned by everyone, forming `UserOp*`. The operation is not valid unless it is completely countersigned. 
+
+Now each party has an effect they want to happen. Isaac wants his payment from Irene. Irene wants her payment from Alice. Alice wants her Tx to be launched on chainC. Each party can force that through by submitting `UserOp*` to the relevant chain (which will cause the relevant wallet to check all the signatures, and slice into the relevant payment to check the chain id).
```

Comment:
After all is said and done, we leave Alice with a deployed wallet on chainC. It probably makes sense for that wallet to be configured with an intermediary, most likely Isaac since he has liquidity on that chain already.